server/src/machines/mod.rs

```diff
@@ -1,7 +1,9 @@
 pub mod dre;
+pub mod extruder1;
 pub mod registry;
 pub mod winder2;
 
 pub const VENDOR_QITECH: u16 = 0x0001;
 pub const MACHINE_WINDER_V1: u16 = 0x0002;
+pub const MACHINE_EXTRUDER_V1: u16 = 0x0003;
```

Comment:
Take a look at the root `README.md` and see that `3` is reserved for Extruder V1 so use `4`.